card/mod/05_standard/set/type/user.rb

```diff
@@ -46,27 +47,25 @@
   # flexibility and security when configuring initial setups
 end
 
-event :setup_first_user, before: :process_subcards, on: :create, when: proc{ |c| Card::Env.params[:setup] } do
-  subcards['signup alert email+*to'] = name
-  subcards['+*roles'] = { content: Card[:administrator].name }
-
-  email, password = subcards.delete('+*account+*email'), subcards.delete('+*account+*password')
-  subcards['+*account'] = { '+*email'=>email, '+*password'=>password }
+event :setup_first_user,
+      before: :process_subcards, on: :create,
+      when: proc { |c| Card::Env.params[:setup] } do
+  add_subcard 'signup alert email+*to', content: name
+  add_subfield :roles, content: Card[:administrator].name
 end
 
-event :signin_after_setup, before: :extend, on: :create, when: proc{ |c| Card::Env.params[:setup] } do
-  Card.cache.delete Auth::NEED_SETUP_KEY
+event :signin_after_setup, before: :extend, on: :create,
+      when: proc { |c| Card::Env.params[:setup] } do
   Auth.signin id
 end
 
 def follow follow_name, option = '*always'
-  if (card = Card.fetch(follow_name)) && (set_card = card.default_follow_set_card)
+  if (card = Card.fetch(follow_name)) &&
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.